includes/pages/user_myshifts.php

```diff
@@ -20,10 +20,19 @@ function user_myshifts()
 {
     $user = auth()->user();
     $request = request();
+    $is_angeltype_supporter = false;
+    if ($request->has('edit')) {
+        $id = $request->input('edit');
+        $shiftEntry = ShiftEntry::where('id', $id)
+            ->where('user_id', User::find($request->input('id'))->id)
+            ->with(['shift', 'user'])
```

Comment:
```suggestion
            ->with(['angelType'])
```
should be enough here but can also be completely omitted as its only one model
```suggestion
```